Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -0,0 +1,18 @@
+<h2>Prevention</h2>
+<h3>Sandmining</h3>
+<p>
+    Do not sand mine, if a player does sand mine the military may be deployed
+    and if it’s severe the criminal may be given a fine of 1g per chunk
+    affected.
+</p>
+<h3>Scams<h3>
+<p>
+    Do not scam people, do trades fairly. If unsure of how much
```

Comment:
Again, `Do not` sounds as if you are talking to someone.

Reply:
ok its been removed

Reply:
It's still there.